_config.yml

```diff
@@ -14,7 +14,8 @@ tutorials_url: /tutorial
 gems:
 - jekyll-seo-tag
 github:
-  repository_url: https://konveyor.github.io/move2kube/
+  repository_url: https://www.github.com/konveyor/move2kube/
   repository_name: Move2Kube
   owner_url: https://konveyor.github.io/
   owner_name: Konveyor 
+  is_project_page: false
```

Comment:
Isn't it a project page?

Reply:
Actually, there were two GitHub buttons on the homepage. The `View on GitHub` button which comes up with the Cayman theme was taking to an incorrect link. I found this https://github.com/pages-themes/cayman/issues/29#issuecomment-339016299. So, to disable that button I have added this line.

Reply:
Okay cool